hw_diag/utilities/hardware.py

```diff
@@ -183,20 +185,44 @@ def set_diagnostics_bt_lte(diagnostics):
     return diagnostics
 
 
+def parse_i2c_bus(address):
```

Comment:
can you write a comment about this function what it does like
"It takes i2c bus as input parameter and extracts the bus number and returns it. "

Reply:
Thanks. Updated!